docs/private-networks/concepts/node-sync-private.md

```diff
@@ -19,14 +19,14 @@ To sync Besu on a private network:
 - Set the network ID and bootnodes specific to your private network.
 - Implement permissioning features to control network access.
 
-Choose the appropriate sync mode based on your private network's requirements and node purposes.
```

Comment:
While "choose" is probably not be the best word, it could still be useful to somewhat guide users toward picking the option that fits their needs. I think the comment related to this was to help users select the option that best works with their setup.